Solve Github CI issues with Windows and TCL

In PR #1238, we encounter CI fails for latest and python 3.12. It says typically:

```
INFO     xtgeo.surface.regular_surface:regular_surface.py:2747 Infotext is <Method: trilinear, window>
=========================== short test summary info ===========================
FAILED tests/test_surface/test_regular_surface_vs_cube.py::test_slice_attr_window_max_w_plotting - _tkinter.TclError: Can't find a usable init.tcl in the following directories: 
    {C:\hostedtoolcache\windows\Python\3.8.10\x64\tcl\tcl8.6}

C:/hostedtoolcache/windows/Python/3.8.10/x64/tcl/tcl8.6/init.tcl: couldn't read file "C:/hostedtoolcache/windows/Python/3.8.10/x64/tcl/tcl8.6/init.tcl": No error
couldn't read file "C:/hostedtoolcache/windows/Python/3.8.10/x64/tcl/tcl8.6/init.tcl": No error
    while executing
"source C:/hostedtoolcache/windows/Python/3.8.10/x64/tcl/tcl8.6/init.tcl"
    ("uplevel" body line 1)
    invoked from within
"uplevel #0 [list source $tclfile]"

This probably means that Tcl wasn't installed properly.
```
This may be a blocker in coming PR' and hence need a quick solution

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -52,6 +52,9 @@ jobs:
       - name: Setup testdata
         uses: "./.github/actions/setup_testdata"
 
+      - name: Set matplotlib backend to avoid potential issues with TCL/TK installation
+        run: echo "MPLBACKEND=Agg" >> $GITHUB_ENV
+
       - name: Run tests
         run: pytest -n 4 tests --disable-warnings --generate-plots
 
@@ -73,7 +76,7 @@ jobs:
 
       - name: Run just hypothesis tests with more examples
         run: pytest -n 4 tests --disable-warnings -m hypothesis --generate-plots
-          
+
   rms:
     name: ${{ matrix.config.name }}
     runs-on: ${{ matrix.config.os }}

